src/components/company/dashboard/ActivityMapGL.test.tsx

```
@@ -0,0 +1,353 @@
+  it('handles click events on markers', async () => {
+    const mockLocations = [
+      { lat: 40.7128, lng: -74.0060, count: 10, trades: ['Electrician'] }
+    ];
+
+    mockUseActivityLocations.mockReturnValue({
+      locations: mockLocations,
+      loading: false,
+      error: null
+    });
+
+    const mockEventHandlers: Record<string, Function> = {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)